src/app/d2-loadout-builder/generated-sets/GeneratedSetButtons.tsx

```diff
@@ -42,22 +42,22 @@ export default function GeneratedSetButtons({
       <button className="dim-button equip-button" onClick={equipItems}>
         {t('LoadoutBuilder.EquipItems', { name: store.name })}
       </button>
-      {t('LoadoutBuilder.ArmorPower', { power: set.power / 5 })}
```

Comment:
We should also remove the `LoadoutBuilder.ArmorPower` key from the locale file @delphiactual is there a way to see if there are any unused i18n items?